integration-cli/check_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/docker/docker/integration-cli/environment"
 	"github.com/docker/docker/integration-cli/fixtures/plugin"
 	"github.com/docker/docker/integration-cli/registry"
+	idaemon "github.com/docker/docker/internal/test/daemon"
```

Comment:
How come we are using an alias for this?
If we must (which would seem a bit off since I don't think we should be importing the real daemon package here), then maybe `testdaemon` is better?

Reply:
yeah `testdaemon` is probably better. We have to use an alias for this right now as this package still refers both. It's in the "Next move"/follow-up PR that I'll try to clean that and migrate all "cli" helpers in `integration-cli/daemon` to api helpers that `internal/test/daemon` package :angel:.
I just didn't want to make a *huge* PR that would be really hard to review...